### PR TITLE
git_task_list.md: Update repository's URL

### LINKED ISSUE #585: Broken link causing failures
http://summerofcode.github.io/gsoc-prep-tasks  moved to https://github.com/coala/git-task-list

### PR DESCRIPTION
**Pull request by RJ722**
Closes #585

### CODE REVIEW

**Comment by RJ722** [comment]
Interesting, because the context of this project is now weird. It's basically pointing towards itself as it's origin:

```markdown
The basic idea is an extended version of [GSOC Task List](https://coala.github.io/git-task-list/).
``` 

What would you suggest @jayvdb?

**Comment by bekicot** [comment]
@RJ722 yea, after the change, it is weird :).

The fix itself looks good to me.

File: _projects/git_task_list.md [hunk 1]
```diff
@@ -22,19 +22,19 @@ tags:
   - community
   - web
 ---
-The basic idea is an extended version of [GSOC Task List](http://github.com/summerofcode/gsoc-prep-tasks).
+The basic idea is an extended version of [GSOC Task List](https://github.com/coala/git-task-list)
```
Comment (pareksha) [inline _projects/git_task_list.md]: Seems you forgot a `.` there. 

And is there really a need to change this URL. They both are directing me to the same page.

Comment (RJ722) [reply _projects/git_task_list.md]: Nice catch!

File: _projects/git_task_list.md [hunk 2]
```diff
@@ -22,19 +22,19 @@ tags:
   - community
   - web
 ---
-The basic idea is an extended version of [GSOC Task List](http://github.com/summerofcode/gsoc-prep-tasks).
+The basic idea is an extended version of [GSOC Task List](https://github.com/summerofcode/gsoc-prep-tasks)
```
Comment (pareksha) [inline _projects/git_task_list.md]: Put a `.` there. :laughing:. Else LGTM

Comment (RJ722) [reply _projects/git_task_list.md]: oops, forgot to push! :-p done!

**Comment by bekicot** [comment]
Looks good to me 👍

**Comment by jayvdb** [comment]
ack 0c4ac45

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**